### PR TITLE
remove Pretender shim

### PR DESCRIPTION
**Pull request by givanse**
~- [ ] depends on the release of Pretender v3.0.0 (pending https://github.com/pretenderjs/pretender/pull/248)~
 - [x] https://github.com/pretenderjs/pretender/pull/255

### CODE REVIEW

**Comment by samselikoff** [comment]
Oh yeah – we haven't been using `shim` since I added auto-import, correct?

**Comment by givanse** [comment]
Correct, ember-auto-import has the magic to make the Pretender IIFE available as if it was an ES6 module. The shim wasn't necessary since then.

With this PR now Pretender can be imported as a native ES6 class, so auto-import is not needed anymore for this dependency (others still need it).

**Comment by samselikoff** [comment]
Ah, very cool!

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "ember-get-config": "^0.2.2",
     "ember-inflector": "^2.0.0 || ^3.0.0",
     "lodash": "^4.17.11",
-    "pretender": "2.1.1"
+    "pretender": "3.0.1"
```
Comment (Turbo87) [inline package.json]: @samselikoff is there a reason why `pretender` is pinned to a specific version? I would like to take advantage of the bug fixes in v3.0.2, but due to the pinning I can't :-/

**Comment by samselikoff** [comment]
@Turbo87 don't believe so, not that I can recall! We can definitely add a ^ there

**Comment by samselikoff** [comment]
mind opening the PR?

**Pull request merged**